salt/vault/secret_backends.sls

```diff
@@ -0,0 +1,37 @@
+enable_transit_secret_backend:
+  vault.secret_backend_enabled:
+    - backend_type: transit
+    - description: Backend to provide encryption, hashing, and randomness as a service
+
+enable_mitx_aws_secret_backend:
+  vault.secret_backend_enabled:
+    - backend_type: aws
+    - mount_piont: aws-mitx
+    - description: Backend to dynamically create IAM credentials
+
+# enable_root_ca_pki_secret_backend:
```

Comment:
Yes, and it will most likely be done as a separate routine. I'll delete the commented out code.